### PR TITLE
upgrade buttons on Ushahidi/platform client.

### PR DESCRIPTION
**Pull request by RachealN**
Fixes Buttons
-Login
-Sign Up

@tuxpiper @noone0212 @Angamanga @rowasc 
Ping @ushahidi/platform

### CODE REVIEW

File: .bashrc [hunk 1]
```diff
@@ -0,0 +1 @@
+export PATH=$HOME/bin:/usr/local/bin:node_modules/.bin:$PATH
```
Comment (rowasc) [inline .bashrc:1]: this should not be here.

**Comment by rowasc** [comment]
Thanks @RachealN ! Left a question and a small comment so far

File: .bashrc (hunk 1 above)
Comment (RachealN) [reply .bashrc:1]: @rowasc, Here's the ticket link https://github.com/ushahidi/platform/issues/3906

Comment (RachealN) [reply .bashrc:1]: > this should not be here.

let me clean it up

Comment (rowasc) [reply .bashrc:1]: ah thanks @RachealN
@noone0212 had sent a PR for a similar issue (or the same) but I see he closed it. @noone0212 can you let us know if you'd want to bring that back?

Comment (noone0212) [reply .bashrc:1]: Yes, @rowasc I would like to bring that back, the issue occurred during rebuilding the platform. I'm so sorry, my bad.

Comment (RachealN) [reply .bashrc:1]: Great
@rowasc can you check this issue https://github.com/ushahidi/platform/issues/3918
i suggested some approach, kindly check it and if you approve, i start to work on it

**Pull request closed without merging**